scripts/xnat/check_new_sessions

```diff
@@ -319,30 +319,36 @@ def incomplete_scan_check(eid, manufacturer, label):
     missing_scans = []
     sess_type = 'xnat:mrSessionData/scans/scan/type'
     scan_types = ifc.select.experiment(eid).attrs.get(sess_type)
+    scan_types_set = set(scan_types)
+    ge_scan_types = set(scan_type_dictionary['GE MEDICAL SYSTEMS'])
+    siemens_scan_types = set(scan_type_dictionary['SIEMENS'])
+
     if manufacturer == 'GE MEDICAL SYSTEMS':
-        for scantype in set(scan_type_dictionary['GE MEDICAL SYSTEMS']):
-            if scantype in set(scan_types):
-                scantype = scantype
-            else:
-                missing_scans.append(scan_types)
+        if ge_scan_types.issubset(scan_types_set):
+            return
+        else:
+            missing_scans = ge_scan_types.difference(scan_types_set)
```

Comment:
is there a way to do this as a set?

`missing_scans = [i for i in scan_types_set if i not in ge_scan_types]`